Documentation/configuration.md

```diff
@@ -149,45 +147,48 @@ In `/etc/rkt/auth.d/specific-tectonic.json`:
 }
 ```
 
-The result is that when downloading data from `kubernetes.io` `rkt`
-still sends `Authorization: Bearer common-token`, but when downloading
-from `coreos.com` - `Authorization: Basic Zm9vOmJhcg==` (`foo:bar`
-encoded in base64). And for `tectonic.com` - `Authorization: Bearer
-tectonic-token`.
+the result is that when downloading data from `kubernetes.io`, `rkt` still
```

Comment:
I don't quite agree but a better way of structuring this escapes me right now so fine :P